Formula/tcpreplay.rb

```diff
@@ -12,23 +12,19 @@ class Tcpreplay < Formula
     sha256 "ab0379428462fbdc2653feae2bcc404cf6b6d2129ddf0461019c53794ce87e4f" => :mavericks
   end
 
-  depends_on "libdnet" => :recommended
+  depends_on :xcode
```

Comment:
This needs tagging with either `=> :run` or `=> :build`. Is this new? Can't remember the need for a full Xcode in the past.